ui/src/views/WorkerManager/routes.js

```diff
@@ -19,7 +19,7 @@ export default path => [
     description: 'View workers for that specific provider',
   },
   {
-    component: WorkerManagerViewWorkerTypes,
+    component: WorkerManagerViewWorkerPools,
     path,
     description:
       'Manage worker types known to the Worker Manager and check on the status of the nodes.',
```

Comment:
I think the description should be update to say "worker pools" instead of "worker types".